src/Exceptions/HttpUnprocessableEntity.php

```diff
@@ -8,4 +8,3 @@ class HttpUnprocessableEntity extends HttpException {
     public const DESCRIPTION = "Means the server understands the content type of the request entity (hence a 415(Unsupported Media Type) status code is inappropriate), and the syntax of the request entity is correct (thus a 400 (Bad Request) status code is inappropriate) but was unable to process the contained instructions.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces